views/partials/header.ejs

```diff
@@ -19,3 +19,19 @@
     </form>
   </div>
 </nav>
+<% Object.entries(messages).forEach(function ([key, value]){
```

Comment:
Can you please investigate if a `switch` is maybe more useful here?
And in my opinion the alert looks better *above* the nav.